activesupport/test/message_encryptor_test.rb

```diff
@@ -106,8 +106,118 @@ def test_messing_with_aead_values_causes_failures
     assert_aead_not_decrypted(encryptor, [text, iv, auth_tag[0..-2]] * "--")
   end
 
-  private
+  def test_metadata_purpose
+    encryptor = ActiveSupport::MessageEncryptor.new(@secret, cipher: "aes-256-gcm")
```

Comment:
We don't need the `:cipher` here, so we should just use the already setup `@encryptor` variable instead of making our own in every test.